docs/administrators/operational_procedures.rst

```diff
@@ -189,6 +189,22 @@ This procedure only works when all agents are autostarted by the server.
    * `autostart_agent_repair_interval`
 
 
+Working directly on the orchestrator
```

Comment:
1. I would move this to it's own section (e.g. 'diagnosing problems`)
2. I would also update the `operational_procedures_with_lsm.html` to use the workon command

Reply:
2. Does inmanta/inmanta-service-orchestrator#356 match your expectations?